docs/admin/backup.rst

```diff
@@ -0,0 +1,177 @@
+Backup and Restore
+==================
+
+.. include:: ../includes/top-warning.rst
+
+QubesOS has a `backup utility <https://www.qubes-os.org/doc/backup-restore/>`_ 
+that allows for backup and restoration of user-specified VMs. 
+
+To perform backups, you will need:
+
+ - an encrypted storage medium
+ - a secure place to store backup credentials    
+
+Backup 
+------
+
+Prepare files in ``dom0``
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Files outside the ``dom0`` home folder aren't normally backed up by the 
+Qubes backup tool. Preserve key configuration files by coping them to the 
+home directory. 
+
+In a ``dom0`` Terminal via **Q ▸ Terminal Emulator**:
+
+  .. code-block:: sh
+
+    mkdir ~/backup/
+    cp -r /usr/share/securedrop-workstation-dom0-config/ ~/backup/
+
+If you have customized :doc:`clipboard access <managing_clipboard>` 
+to SecureDrop Workstation, or have made any other customizations 
+involving RPC policies, back up related configuration files:   
```

Comment:
In my experience, tags are restored when a qube is restored. (I remembered saying this [a while back](https://github.com/freedomofpress/securedrop-workstation/issues/120#issuecomment-656918934), but I was doubting myself, so I just tried tagging and restoring a test qube and indeed the custom tags were still present).  

Backing up the RPC policy directories (with the idea that admins could audit them if need be, but not restore them directly) was some sort-of-futureproofing in case we are dealing with customized export workflows that involve customized qubes. But I can remove it and we can cross that bridge if/when we come to it.

Reply:
I've removed all of this in 05b8dfd and left a note that advanced users may need to back up additional `dom0` components. I think for now, that's a reasonable tradeoff information-wise. If we have orgs using a lot of customized qubes and/or modifying RPC policies to accommodate their own workflows down the line, we can address this in more depth (or maybe we will develop a system that avoids the need for them doing this).